packages/matter.js/src/protocol/ControllerCommissioner.ts

```diff
@@ -116,7 +117,7 @@ export class CommissioningError extends MatterError {}
 /** Error that throws when Commissioning fails but process can be continued. */
 class RecoverableCommissioningError extends CommissioningError {}
 
-const DEFAULT_FAILSAFE_TIME_MS = 60000;
+const DEFAULT_FAILSAFE_TIME_MS = 60_000; // 60 seconds
```

Comment:
Maybe we could rename 60_000 to ONE_MINUTE_MS

Reply:
:-)) downside is that you then require other files again ... or you have it duplicated ... Would leave for now :-)